common-content/en/module/js1/reusing-variable-names/index.md

```diff
@@ -0,0 +1,195 @@
++++
+title = '🔄 Reusing variable names'
+
+time = 30
+facilitation = false
+emoji= '🔄'
+[objectives]
+1='Interpret an error message'
+2='Break down an error message and look up unfamiliar parts'
+3='Rephrase an error message'
+[build]
+  render = 'never'
+  list = 'local'
+  publishResources = false
+
++++
+
+We saw this error - let's try to understand it:
+
+```
+SyntaxError: Identifier 'currentOutput' has already been declared
+```
+
+### Knowing what we changed
+
+It can be useful to remember when our code last worked, and what we changed since then.
+
+When we just had the first 10 lines of code here, everything worked, but we got this error when we added the rest:
+
+```js {linenos=table,linenostart=1,hl_lines=["12-17"]}
+function formatAs12HourClock(time) {
+  return `${time} am`;
+}
+
+const currentOutput = formatAs12HourClock("08:00");
+const targetOutput = "08:00 am";
+console.assert(
+  currentOutput === targetOutput,
+  `current output: ${currentOutput}, target output: ${targetOutput}`
+);
+
+const currentOutput = formatAs12HourClock("23:00");
+const targetOutput = "11:00 pm";
+console.assert(
+  currentOutput === targetOutput,
+  `current output: ${currentOutput}, target output: ${targetOutput}`
+);
+```
+
+{{<note type="tip" title="Tip">}}
+
+This is one of the reasons it's useful to run our code often.
+
+If we only changed one small thing since our code last worked, it's easier to find the problem. If we wrote 100 lines of code since we last saw it work, it will be harder to identify which change was the problem.
```

Comment:
```suggestion
If we changed one thing since our code last worked, we know what change is the problem. If we have written 100 lines of code since we last saw it work, we have 100 possible problems.
```